### ISSUE TITLE
Request Prob 140 Spring 2020 files

### ISSUE THREAD

**Issue by fan005** [berkeley-dsep-infra/hubs#3443]
### Email Id

lz0417

### Link to Datahub Folder

gs://ucb-datahub-archived-homedirs/spring-2020/prob140/lz0417.tar.gz

**Comment by felder** [comment]
Please do not attempt to construct archived file locations.

Instead use the location placed in your account:

```
/prob140/lz0417# cat WHERE-ARE-MY-FILES.txt

Your files have been archived due to inactivity.

If you want to retrieve a copy of your files, please
open a Data Archival Request via github here:
https://github.com/berkeley-dsep-infra/datahub/issues/new?assignees=&labels=support&template=data_archival_request.yml

The following text is the link to your datahub folder,
it must be included with your request:

gs://ucb-datahub-archived-homedirs/2022-1-spring/prob140/lz0417.tar.gz
```

**Comment by felder** [comment]
Email ID is: lz0417@berkeley.edu

**Comment by felder** [comment]
Signed URL sent!

**Comment by fan005** [comment]
Hi there! I actually have no data in Prob 140 Spring 2020 but I do in Prob 140 Spring 2020. I tried to have an archive for all my courses since I graduate. Is it too late to request a data retrieval since it has been two years?

**Comment by felder** [comment]
@fan005 

_I actually have no data in Prob 140 Spring 2020 but I do in Prob 140 Spring 2020_

Can you rephrase this please?

**Comment by fan005** [comment]
Sincere apology. I meant to say I have no data in Spring 2022, but I have data in Spring 2020. The link you sent me seems to be the one for Spring 2022.

**Comment by felder** [comment]
@fan005 Did you download and extract the archive to see if your files were there?

**Comment by fan005** [comment]
Hi there! I unzip the downloaded tar file and get a 400Mb file (the file type is simply File). But I'm not sure what I can do with it...

**Comment by felder** [comment]
The file in question is a tar archive that is compressed with gzip.  You'll want to download the file to your personal computer and extract it.

If your computer is a Mac, the functionality should be built in and you should be able to just double click it.  If your computer is a PC, you'll probably need a 3rd party tool to open it.  I'd recommend 7-zip for that.  It's free and it'll open just about any archive/compression format you're likely to encounter.

https://www.7-zip.org/

Once you have the files extracted, you can decide what specifically you want to keep and back it up however you see fit.

**Comment by fan005** [comment]
Hi there. I have already used 7zip to unzip the tar file and ended up getting a single large file that is in file type File...

**Comment by felder** [comment]
@fan005 add .tar to end of the large file and open that in 7zip.

Basically it is archived and then compressed, you need to decompress it and then extract the contents of the archive.

**Comment by fan005** [comment]
Now it works! Thank you!

**Comment by felder** [comment]
Ok, good!
